receiver/apachereceiver/documentation.md

```diff
@@ -26,8 +26,8 @@ metrics:
 
 ## Metric attributes
 
-| Name | Description |
-| ---- | ----------- |
-| scoreboard_state | The state of a connection. |
-| server_name | The name of the Apache HTTP server. |
-| workers_state | The state of workers. |
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| scoreboard_state (state) | The state of a connection. | open, waiting, starting, reading, sending, keepalive, dnslookup, closing, logging, finishing, idle_cleanup |
+| server_name | The name of the Apache HTTP server. |  |
+| workers_state (state) | The state of workers. | busy, idle |
```

Comment:
I think it would be preferable to only list the "value", when it is specified. In such cases, the "name" is really just a developer facing key that allows us to differentiate between two attributes with the same user facing value. 

```suggestion
| state | The state of workers. | busy, idle |
```

Reply:
The value of the developer-facing key in documentation is to connect the list of attributes next to the metrics to the attribute documentation. If there are two "type" attributes this allows you to connect them to the correct metric.

Reply:
Ah, I see.